index.vwf.html

```diff
@@ -30,11 +30,24 @@
 <script type="text/javascript" src="source/view/videoController.js"></script>
 <script type="text/javascript" src="source/view/fileManager.js"></script>
 
+
 <!-- CSS -->
 <link rel="stylesheet" href="source/view/index.css"></link>
 <link rel="stylesheet" href="source/view/jquery-resizable.css"></link>
+<link type="text/css" href="minimal_jplayer_style.css" rel="stylesheet" />
 
 <!-- HTML -->
+<div id="jp_container_1" class="jp-video jp-video-360p">
+
+    <!--container in which our video will be played-->
+    <div id="jquery_jplayer_1" class="jp-jplayer"></div>
+ 
+    <!--main containers for our controls-->
+    <div class="jp-gui">
+        <div class="jp-interface">
+        </div>
+    </div> 
+</div>        
 <div id="version"></div>
```

Comment:
I'm sure in both cases this was done to save time.  As long as it's not on
development then I'm ok with it.  It just needs to be turned into a string
with those ids externalized to the options.  That way the driver can insert
the html at the bottom of the body.

On Fri, Feb 13, 2015 at 9:40 AM, Nathan Marshak <notifications@github.com>
wrote:

> In index.vwf.html
> <https://github.com/virtual-world-framework/mars-game/pull/493#discussion_r24667595>
> :
>
> >
> >  <!-- HTML -->
> > +<div id="jp_container_1" class="jp-video jp-video-360p">
> > +
> > +    <!--container in which our video will be played-->
> > +    <div id="jquery_jplayer_1" class="jp-jplayer"></div>
> > +
> > +    <!--main containers for our controls-->
> > +    <div class="jp-gui">
> > +        <div class="jp-interface">
> > +        </div>
> > +    </div>
> > +</div>
> >  <div id="version"></div>
>
> @scottnc27603 <https://github.com/scottnc27603> Right now there's a
> dependency on the correct divs in index.vwf.html. This is also true of TDG,
> however, which uses the current, upstream jPlayer driver. I do not see this
> as a regression.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/virtual-world-framework/mars-game/pull/493/files#r24667595>
> .
>